src/ripple/protocol/Rate.h

```diff
@@ -30,8 +30,9 @@ namespace ripple {
 
 /** Represents a transfer rate
 
-    Transfer rates are specified as fractions of 1 billion. For example, a
-    transfer rate of 1% is represented as 1010000000.
+    Transfer rates are specified as fractions of 1 billion.
+    For example, a transfer rate of 1% is represented as
+    1010000000.
```

Comment:
It would be great to add a note here about the valid range of values for a `Rate`.  In particular, I believe that `0` is considered an invalid value.